e2e/specs/results-viewer.spec.ts

```diff
@@ -47,53 +47,46 @@ test('Record and edit test result', async ({ page }) => {
     await page.getByRole('button', { name: 'Hematology', exact: true }).click();
   });
 
-  await test.step('And I fill the complete blood count form', async () => {
+  await test.step('And I fill the "Complete Blood Count" section', async () => {
+    await page.locator('#ManualInputWhiteBloodCellsid').clear();
```

Comment:
I think using clear() method should be applicable when editing data otherwise the field is blank.

Reply:
True!